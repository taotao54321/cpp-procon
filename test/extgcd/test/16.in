999999208 999999337
