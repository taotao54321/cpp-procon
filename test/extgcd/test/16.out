449612105 -449612047
